baseplate/clients/memcache/__init__.py

```diff
@@ -153,14 +153,14 @@ class MemcacheContextFactory(ContextFactory):
 
     def __init__(self, pooled_client: PooledClient, name: str = "default"):
         self.pooled_client = pooled_client
-
-        pool = self.pooled_client.client_pool
-        self.pool_size_gauge.labels(name).set_function(lambda: pool.max_size)
-        self.free_connections_gauge.labels(name).set_function(lambda: len(pool.free))
-        self.used_connections_gauge.labels(name).set_function(lambda: len(pool.used))
+        self.name = name
 
     def report_memcache_runtime_metrics(self, batch: metrics.Client) -> None:
```

Comment:
Do you know how this is used? I don't see this called anywhere in baseplate.py

Reply:
https://github.com/reddit/baseplate.py/blob/bffa1d0c65a8764fe76235cdf946410a2e67dd93/baseplate/__init__.py#L530-L545 :magic_wand:

Reply:
The runtime monitor https://github.com/reddit/baseplate.py/blob/297cc8597770e497364d715b047e1be872d03944/baseplate/server/wsgi.py#L57

Runs this thread
https://github.com/reddit/baseplate.py/blob/48bcb0b099608aa8b8eecbf97cb929537d68753d/baseplate/server/runtime_monitor.py#L187-L215

Getting reporters https://github.com/reddit/baseplate.py/blob/48bcb0b099608aa8b8eecbf97cb929537d68753d/baseplate/server/runtime_monitor.py#L249

From here https://github.com/reddit/baseplate.py/blob/bffa1d0c65a8764fe76235cdf946410a2e67dd93/baseplate/__init__.py#L530-L545

.. i think.

runtime_monitor.py has some interesting things we probably also want to ship to prometheus